base/ca/src/main/java/com/netscape/cms/profile/common/EnrollProfile.java

```diff
@@ -373,9 +375,10 @@ public void setDefaultCertInfo(Request request) throws EProfileException {
             info.set(X509CertInfo.KEY,
                     new CertificateX509Key(X509Key.parse(new DerValue(dummykey))));
 
-            info.set(X509CertInfo.SUBJECT,
-                    new CertificateSubjectName(new X500Name("")));
-
+            if (configStore.getBoolean("ca.emptySubject", true)) {
```

Comment:
I think you should add a dummy name in case of false to restore the previous behaviour ([as a reference](https://github.com/edewata/pki/blob/8c72966ebfe0ff8984b6ee0860e524bf5fa89c9c/base/server/src/com/netscape/cms/profile/common/EnrollProfile.java#L302)) needed by CC.

Reply:
Oops, the version I was working off was just blank, thanks!